src/uct/ib/base/ib_md.c

```diff
@@ -1580,20 +1580,29 @@ ucs_status_t uct_ib_md_open(uct_component_t *component, const char *md_name,
     return status;
 }
 
-void uct_ib_md_parse_relaxed_order(uct_ib_md_t *md,
-                                   const uct_ib_md_config_t *md_config)
+static void uct_ib_md_parse_relaxed_order(uct_ib_md_t *md,
+                                          const uct_ib_md_config_t *md_config,
+                                          size_t mem_size, size_t mr_size)
```

Comment:
IMO the logic of calculating mr size should be in another function, like
uct_ib_md_init_memh_size(md, memh_base_size, mr_size)